common/src/main/java/com/wynntils/models/character/CharacterModel.java

```diff
@@ -45,7 +46,7 @@ public final class CharacterModel extends Model {
     // we need a .* in front because the message may have a custom timestamp prefix (or some other mod could do
     // something weird)
     private static final Pattern WYNN_DEATH_MESSAGE = Pattern.compile(".*§r §4§lYou have died\\.\\.\\.");
-    private Vec3 lastLocationBeforeTeleport;
+    private Position lastLocationBeforeTeleport;
```

Comment:
Can you also please rename it so the name is "Position" instead of "Location"? I've tried to make sure the code base follows this pattern to help minimize confusion (once you learn about the difference, at least...)

Reply:
Then you are done and we can integrate. Thanks for hanging in there, I realize there is a lot of coding differences between Legacy and Artemis!